bin/gwdetchar-omega

```diff
@@ -330,7 +330,10 @@ for block in blocks[:]:
 
         # filter the timeseries
         corner = c.frange[0] / 1.5
-        hpseries = series.highpass(corner, gpass=.5, gstop=100, filtfilt=True)
+        wn = 2 * corner * series.dt.decompose().value
+        filt = signal.butter(12, wn, btype='highpass', analog=False,
+                             output='zpk')
```

Comment:
@alurban, why not use `output='sos'` directly?

Reply:
@duncanmmacleod, oops! I didn't update this after you pointed out `gwdetchar` explicitly requires `scipy >= 0.16.0`, I can update it now